Mathlib/Data/List/Range.lean

```diff
@@ -221,6 +221,9 @@ theorem finRange_map_get (l : List α) : (finRange l.length).map l.get = l :=
   List.ext_get (by simp) (by simp)
 #align list.map_nth_le List.finRange_map_get
 
+lemma finRange_map_comp_get : (finRange l.length).map (f ∘ l.get) = map f l := by
```

Comment:
`f` and `l` should be explicit here.

Reply:
The above has `l` implicit. Should it be explicit only here?